<?xml version="1.0" encoding="utf-8"?>
<launch>
  <arg name="computer_name" value="usl" />
  <param name = "/use_sim_time" value="true"/>

  <node  pkg="checkerboard_plane_detection" name="checkerboard_plane_detection_os" type="checkerboard_plane_detection_node" output="screen">
	<remap from="/cloud_in" to="/os1_cloud_node/points/rotated"/>
        <remap from="/plane_out" to="/os1_cloud_node/points/plane"/>
  	<param name = "x_min" type= "double" value= "0.75"/>
  	<param name = "x_max" type= "double" value= "5"/>
  	<param name = "y_min" type= "double" value= "-1.5"/>
  	<param name = "y_max" type= "double" value= "1.5"/>
  	<param name = "z_min" type= "double" value= "-1.5"/>
  	<param name = "z_max" type= "double" value= "3"/>
  	<param name = "min_pts" type= "int" value= "500"/>
  	<param name = "ransac_threshold" type= "double" value= "0.01"/>
  	<param name = "remove_outlier" value="true"/>
  	<param name ="target_config_file_path" value="/home/$(arg computer_name)/catkin_ws/src/camera_lidar_calibration/config/target_3.yaml"/>
  </node>
</launch>
